### ISSUE TITLE
Exploding Messages animation disable

### ISSUE THREAD

**Issue by JeremyRubin** [keybase/client#17137]
It would be great if the exploding messages animation can be made less fun (e.g., a simple fade away) rather than a flashing color swipe. When I have a conversation, I have a bunch of messages within 30s of each other, so it basically makes my screen flash repeatedly. This is fun at first, but ultimately it's distracting and it makes me look at keybase more than necessary. I end up leaving Keybase minimized, which makes it less useful of a work tool.

As a general principal, I think that the UX should only noisily update when it isn't the focused window when there is something that I should see (e.g., a notification).

**Comment by mart-w** [comment]
Good idea. When the application is currently focused, I see no problem with the animations being a bit more playful. But when it’s only running on the side, it really shouldn’t draw more attention than needed.
